spec.html

```diff
@@ -3236,7 +3236,7 @@ <h2>[[OwnPropertyKeys]] ( )</h2>
             The returned List must not contain any duplicate entries.
           </li>
           <li>
-            The Type of each element of the returned List is either String or Symbol.
+            Each element of the returned List is a property key.
```

Comment:
```suggestion
            Each element of the returned List must be property key.
```
The status quo is an "is", but the neighboring points are "musts", so not sure.

Reply:
This is true, but on 15901 we have the exact same sentence. So maybe the other "musts" should change? Or maybe it doesn't matter that we're consistent with them.

Reply:
I changed this one, happy to change them back if you want